src/open_inwoner/openzaak/clients.py

```diff
@@ -680,9 +693,13 @@ def fetch_open_tasks(self, bsn: str) -> list[OpenTask]:
         return results
 
 
-def _build_zgw_client(type_) -> APIClient | None:
+ZgwClientType = Literal["zaak", "catalogi", "document", "form"]
+ZgwClient: TypeAlias = ZakenClient | CatalogiClient | DocumentenClient | FormClient
+
+
+def _build_zgw_client(type_: ZgwClientType) -> ZgwClient | None:
     config = OpenZaakConfig.get_solo()
-    services_to_client_mapping = {
+    services_to_client_mapping: Mapping[ZgwClientType, ZgwClient] = {
```

Comment:
There is a mismatch between annotation and assignment: the values in the dictionary are of type `Type`, but what's expected according to annotation is a `Union` of clients.